pages/manage/productions/index.vue

```diff
@@ -4,9 +4,9 @@
       <pub-page-title>
         {{ $t('menu.productions') }}
         <small>
-          <nuxt-link class="link" to="productions/create">{{
-            $t('productions.overview.create')
-          }}</nuxt-link>
+          <nuxt-link to="productions/create">
+            <pub-link>{{ $t('productions.overview.create') }}</pub-link>
+          </nuxt-link>
```

Comment:
Wrapping `pub-link` in `nuxt-link` results in a double `a` tag in the final HTML:

![Screenshot 2020-09-11 at 10 12 05](https://user-images.githubusercontent.com/959026/92890339-55417600-f417-11ea-91d1-e3f4cf4cd468.png)

I think you could fix it with an approach like this:
```
<pub-link :as="nuxt-link" to="productions/create">{{ $t('productions.overview.create') }}</pub-link>
```

To make that `:as` work, see https://medium.com/bitbond/vue-load-dynamic-components-based-on-a-string-variable-52f54860a0d